examples/basic/android/app/src/main/java/com/videoplayer/MainApplication.java

```diff
@@ -25,11 +21,12 @@ public boolean getUseDeveloperSupport() {
 
     @Override
     protected List<ReactPackage> getPackages() {
-      return Arrays.asList(
```

Comment:
OK I double check, the sample works correctly.

So the autolink seems to be done automatically with last react-native version.
can you just patch the README.md to indicate the change in following section ?

#### **MainApplication.java**